### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by WilliamDavidHarrison**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,10 @@ A customisable, flexible and, self hostable flat file CMS for bio link lists.
 - [Website](https://cmshark.com)
 - [Documentation](https://docs.cmshark.com)
 - [Demo](https://cmshark.com/demo)
-- [Working Example](https://will.clarke.ml)
+
+### Working Examples
+- [Working Example 1](https://will.clarke.ml)
+- [Working Example 2](https://williamharrison.me)
```
Comment (WilliamDavidHarrison) [inline README.md:13]: Remove this is you want and just move it to links

**Comment by WilliamDavidHarrison** [comment]
@wclarkey Why did you close it?

**Comment by wclarkey** [comment]
unnecessary changes

**Comment by WilliamDavidHarrison** [comment]
> unnecessary changes

True, but it makes it look nicer

**Pull request closed without merging**